cypress/e2e/3-cypress-assertions/3-error-messages/email-adress/empty.cy.js

```diff
@@ -0,0 +1,11 @@
+describe('Check error messages for each fields', ()=>{
+    context(' page -  /auth/register', ()=>{
+
+        it('Error behind "Email address"(empty input)',()=>{ 
+        cy.visit('/auth/register');
+        cy.get('[ng-reflect-status]').eq(1).click()
+        cy.get('[ng-reflect-status]').eq(2).click()
```

Comment:
This part is the same for a few specs. You can create a function for it, and call where you need.